src/device/test/Dual_Call.cu

```diff
@@ -0,0 +1,87 @@
+///----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   device/test/Dual_Call.cu
+ * \author Alex R. Long
+ * \date   Mon Mar 25 2019
+ * \brief  Show how code can be called from GPU and host
+ * \note   Copyright (C) 2019 Triad National Security, LLC.
+ *         All rights reserved.
+ */
+//---------------------------------------------------------------------------//
+
+#include "Dual_Call.hh"
+
+namespace rtt_device_test {
+
```

Comment:
Please add doxygen-style comment blocks for each function.

Reply:
Done. Thanks!